### ISSUE TITLE
https://github.com/dequelabs/axe-core/blob/v3.1.2/doc/rule-descriptions.md#css-orientation-lock contains incorrect WCAG reference.

### ISSUE THREAD

**Issue by jeankaplansky** [dequelabs/axe-core#1345]
Expectation: Name proper WCAG Criterion

Actual: Names improper WCAG Criterion:

Affected file: https://github.com/dequelabs/axe-core/blob/v3.1.2/doc/rule-descriptions.md, tag 3.1.2

css-orientation-lock | Ensures content is not locked to any specific display orientation, and the content is operable in all display orientations | Serious | cat.structure, wcag262, wcag21aa, experimental | true
-- | -- | -- | -- | --

There is no WCAG 262. This should be marked wcag21aa 1.3.4. Please remove the wcag262 and replace with wcag134.

**Comment by WilcoFiers** [comment]
Yikes! OK, thank you Jean. @jkodu this one's for you :)

**Comment by jeankaplansky** [comment]
You’re welcome!!!

**Comment by dylanb** [comment]
@wilco @jkodu this is critical for 3.2.0

**Comment by dylanb** [comment]
Its not just the documentation, its the actual tag that is incorrect
